Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -279,7 +279,7 @@
   - type: Pda
     id: MimeIDCard
     state: pda-mime
-    idSlot:
+    idSlot: #  rewrite without sound because mime
```

Comment:
pretty sure you can just set the sound datafield to null

Reply:
i looked and thought "why is it writted again?"
so its just a note for that ones who will think the same